controllers/controller.js

```diff
@@ -1,4 +1,8 @@
-const { selectTopics, selectAuthorById } = require("../models/model");
+const {
+  selectTopics,
+  selectAuthorById,
+  updateArticlesById,
+} = require("../models/model");
```

Comment:
You should have SEPARATE files for topics and authors models. Have a look at my 'naming conventions' file that I posted in the cohort channel yesterday.